docs/how-to/tour.rst

```diff
@@ -0,0 +1,122 @@
+Tour of the Code
+================
+
+Before getting started, it's nice to know a bit about how the project is structured and where
+to look for examples. This document aims to provide a brief tour of the
+important features of the code. It's aimed at new contributors, but frequent flyers can
+skip down to the bottom for a quick reference.
+
+General Structure
+-----------------
+
+Core Code
+*********
+
+Batavia implements the Python native types, builtins, and standard library partially in JavaScript.
```

Comment:
This could be a a little clearer - "the core language (types, builtins and the core interpreter) are all javascript, as well those portions of the standard library that depend on system services".